src/Core/Grid/Definition/Factory/TitleGridDefinitionFactory.php

```diff
@@ -0,0 +1,238 @@
+                    ->setName($this->trans('Title', [], 'Admin.Global'))
+                    ->setOptions([
+                        'field' => 'name',
+                    ])
+            )
+            ->add(
+                (new DataColumn('type'))
+                    ->setName($this->trans('Gender', [], 'Admin.Global'))
+                    ->setOptions([
+                        'field' => 'type',
+                        'sortable' => false,
+                    ])
+            )
+            ->add(
+                (new ImageColumn('image'))
+                    ->setName($this->trans('Image', [], 'Admin.Global'))
+                    ->setOptions([
+                        'src_field' => 'image',
+                    ])
+            )
+            ->add(
+                (new ActionColumn('actions'))
+                    ->setName($this->trans('Actions', [], 'Admin.Global'))
+                    ->setOptions([
+                        'actions' => (new RowActionCollection())
+                            ->add(
+                                (new LinkRowAction('edit'))
+                                    ->setName($this->trans('Edit', [], 'Admin.Actions'))
+                                    ->setIcon('edit')
+                                    ->setOptions([
+                                        'route' => 'admin_title_edit',
+                                        'route_param_name' => 'titleId',
+                                        'route_param_field' => 'id_gender',
+                                        'clickable_row' => true,
+                                    ])
+                            )
+                            ->add(
+                                $this->buildDeleteAction(
+                                    'admin_title_delete',
+                                    'titleId',
+                                    'id_gender',
+                                    Request::METHOD_DELETE
+                                )
+                            ),
+                    ])
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getFilters()
+    {
+        return (new FilterCollection())
+            ->add(
+                (new Filter('id_gender', NumberType::class))
+                    ->setTypeOptions([
+                        'required' => false,
+                        'attr' => [
+                            'placeholder' => $this->translator->trans('Search ID', [], 'Admin.Actions'),
+                        ],
+                    ])
+                    ->setAssociatedColumn('id_gender')
+            )
+            ->add(
+                (new Filter('name', TextType::class))
+                    ->setTypeOptions([
+                        'required' => false,
+                        'attr' => [
+                            'placeholder' => $this->translator->trans('Search title', [], 'Admin.Shopparameters.Feature'),
```

Comment:
```suggestion
                            'placeholder' => $this->translator->trans('Search title', [], 'Admin.Global'),
```

Just in case, it could be used elsewhere in the software.

Reply:
Fixed :)

Reply:
Thanks!